### DISCUSSION TITLE
CUDA OOM [Copy from Wechat Group]

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmediting#1285] [Models] [answered]
请问一下，尝试使用mmediting的realbasicvsr跑一个视频超分的demo，然后报错CUDA OOM，是我的输入视频太大吗……可是我的视频一共只有120帧。

![image](https://user-images.githubusercontent.com/25839884/192181642-8926216c-63dc-400c-aaa1-804f4323f143.png)

**Comment by zengyh1900** [comment]
Please check this issue. @Z-Fran

**Comment by Z-Fran** [comment, marked as answer]
Please check GPU memory. Realbasicvsr的demo显存最少需要8G.
